Doc/library/pdb.rst

```diff
@@ -570,7 +570,16 @@ can be overridden by the local file.
 
    Start an interactive interpreter (using the :mod:`code` module) whose global
    namespace contains all the (global and local) names found in the current
-   scope.
+   scope. Use ``exit()`` or ``quit()`` to exit the interpreter and return to
+   the debugger.
+
+   .. note::
+
+      Because interact creates a new global namespace with the current global
+      and local namespace for execution, assignment to variables will not
+      affect the original namespaces.
+      However, modification to the mutable objects will be reflected in the
+      original namespaces.
```

Comment:
Should we have a versionchanged entry here?

Reply:
Which change do you want to refer here? The notes part is just an explanation for existing behavior. ``exit()`` and ``quit()`` are new - the users used to have one option - EOF to exit from `interact` command. Do you think the availability for `exit()` and `quit()` grants a version changed entry? Or you are takling about this PR? The fact that `interact` command outputs to the same channel as `pdb` now? We an do that as well if you think that should be done.

Reply:
Okay added both in the docs.